CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/Models/ChatDto.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CheckDrive.Mobile.Models
+{
+    public class ChatDto
+    {
+        public string HeadlineText { get; set; }
+        public string Title { get; set; }
+    }
+}
```

Comment:
I think it would be better to put <;> after the namespace CheckDrive.Mobile.Models